GVRf/IO/ControllerSample/assets/gvr_note4.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
both this and the other one are using the old format.  in fact, you can get rid of the gvr_s5 one.  please copy the one from simple-sample.